package.json

```diff
@@ -58,30 +58,34 @@
   ],
   "license": "ISC",
   "dependencies": {
-    "chownr": "^1.1.1",
+    "chownr": "^1.1.2",
     "figgy-pudding": "^3.5.1",
+    "fs-minipass": "^1.2.7",
     "glob": "^7.1.4",
-    "graceful-fs": "^4.1.15",
-    "infer-owner": "^1.0.3",
+    "graceful-fs": "^4.2.2",
+    "infer-owner": "^1.0.4",
     "lru-cache": "^5.1.1",
-    "mississippi": "^3.0.0",
+    "minipass": "^2.6.2",
+    "minipass-collect": "^1.0.1",
+    "minipass-flush": "^1.0.3",
+    "minipass-pipeline": "^1.1.1",
     "mkdirp": "^0.5.1",
     "move-concurrently": "^1.0.1",
     "p-map": "^3.0.0",
     "promise-inflight": "^1.0.1",
-    "rimraf": "^2.6.3",
-    "ssri": "^6.0.1",
+    "rimraf": "^2.7.1",
+    "ssri": "github:npm/ssri",
```

Comment:
🤔 is this temporary?

Reply:
Yes :)

Before this can land (or at least, before it can be published) we'll need to merge in ssri's v7 branch, and publish the new semver major.

Reply:
See: https://github.com/npm/ssri/pull/3

Reply:
Yes, hopefully it won't stay that way for long :)

https://github.com/npm/ssri/pull/3